website/blog/2023-lcf/2023-02-01/02_YouSayLowISayCode.md

```diff
@@ -0,0 +1,70 @@
+---
+slug: 2023-day2
+title: You say Low, I say Code!
+authors: [april, matt, sonja, nitya]
+tags: [low-code-february, 30-days-of-lowcode, learn-live, zero-to-hero, ask-the-expert,fusion-teams, power-platform]
+---
+## You say Low, I say Code!
+
+There is a low code revolution underway. We have a surging digital demand with an estimated 500 million more apps that will need to be created in the next 5 years. More and more companies are leveraging low code and no code platforms to bridge this app gap and address workflow and automation needs. In fact, Gartner estimates that 65% of application development will be low code by 2024.
+
+With low code tools like the Power Platform making it easier for business technologists to develop solutions, you might wonder if there's any room for a code first developer in this low code revolution. Well, the answer is, absolutely there is! In fact, code first developers are a critical piece to the puzzle.
+
+## Low Code + Full Code = No Cliffs
+
+Using low-code tools is key to helping to resolve the existing and impending app gap challenge but that doesn't mean that low code alone can solve every business problem. Low code tools like the Power Platform's full potential is unlocked when you combine the low code capabilities with code first development.
+
+The combination of low code + full code creates what we call a "no cliffs" experience that allows business technologists to easily build solutions with low code while professional developers can leverage full code to augment and extend the platform capabilities. The process of low code developers, IT pros and full code developers working together like this is often referred to as "fusion development".
+
+![Fusion Development Process](./02_FusionDevProcess.png)
+
+## What is "low code"?
+
+The term "low code" can mean different things to different people when they first hear it. What we mean when we talk about low code is that with tools like Power Apps, you only need to write a small amount of code to get results that would normally take several more lines of code in a traditional programming language.
+
+![What is Low Code](./02_WhatIsLowCode.png)
+
+In canvas-based Power Apps, the low code scripting language used is called [Power Fx](https://aka.ms/LearnPowerFx). This language is based on the Microsoft Excel formula language which meets many business technologists where they are by leveraging the knowledge they already have.
```

Comment:
Would it make sense to put this section above the previous? So we'd explain right away what low code is?

- You say low, I say code
- What is low code
- Low code + full code = no cliffs

Reply:
Great suggestion Matt. I moved it up in the blog